Gauge/Internal.hs

```diff
@@ -71,8 +71,8 @@ runOne desc bm file = do
       Just prog -> gaugeIO $ runBenchmarkWith prog desc timeLimit quickMode
       Nothing -> gaugeIO $
         if quickMode
-        then runBenchmark bm threshold 2 0
-        else runBenchmark bm threshold 10 timeLimit
+        then runBenchmark bm 30 2 0
```

Comment:
I don't think it's a good idea to remove the self-explanatory binding `threshold` in favor of hardcoding the value in 3 different places with no explanation. it would have been easier to move to internal as a constant relative to measurement instead of analysis.

Reply:
I thought quite a bit when writing this and then finally discarded the binding. The reasoning was if I am using a binding for threshold then I should use a binding for number of samples as well (the other hardcoded argument). When I used that as well I thought I am adding too much for too little gain because the function definition explains the arguments and if we type them better later then it will become obvious.

Also, this is the only place where we would be using it. The other place i.e. `getOverhead` is an ugliness that should go away very soon.